fdbclient/BulkLoading.cpp

```diff
@@ -86,25 +86,48 @@ std::string generateBulkLoadJobManifestFileContent(const std::map<Key, BulkLoadM
 	return res;
 }
 
+// For now, we only support blobstore:// urls. Later, lets add support for file:// urls, etc.
+// 'blobstore://' is the first match, credentials including '@' are optional and second regex match.
```

Comment:
Can you use TODO(BulkLoad): XXX here, so that we can get those tasks by searching "TODO(BulkLoad)" globally.

Reply:
done